routes/index.js

```diff
@@ -19,6 +19,10 @@ router.get('/', function (req, res, next) {
     }
   }
 
+  if (req.query.errorMsg) {
+    responseData.errorMsg = req.query.errorMsg
+  }
```

Comment:
Passing around error "tokens" and then turning them into human error messages might be good practice, but a bit cumbersome for this workshop.

Reply:
OK. Was thinking of letting facilitators do that as a refactor. But yeah, this is fine.